app/src/main/java/com/parinaz/weblog/MainActivity.kt

```diff
@@ -2,10 +2,27 @@ package com.parinaz.weblog
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.recyclerview.widget.RecyclerView
+import com.parinaz.weblog.adapter.PostAdapter
+import com.parinaz.weblog.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+
+    lateinit var binding: ActivityMainBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
```

Comment:
Convert binding into a class member variable instead of a local variable

Reply:
done